games/i.yaml

```diff
@@ -170,8 +169,7 @@
   - Iron Seed
   repo: http://www.ironseed.com/ironseed-v1.20.0016-2013-03-17.zip
```

Comment:
this is also gone - let's remove the whole game

Reply:
you could still access it through the archive https://web.archive.org/web/20150725164943/http://www.ironseed.com/ironseed-v1.20.0016-2013-03-17.zip
delete or keep?

Reply:
ok; linking to the web archive is fine too

Reply:
did it